waitress/app/forms.py

```diff
@@ -1 +1,18 @@
-from django.forms import ModelForm
+from django.forms import Form, CharField, PasswordInput, TextInput
+
+
+class LoginForm(Form):
```

Comment:
Why django form? are we moving to server side rendered apps?

Reply:
Because they provide validation out of the box for me.

Reply:
Do you have a suggestion for this? I used django forms because they provided me with validation and didn't want to write javascript to do that at this point.

Reply:
My comment was basically why we are not making xhr request for the form submission rather than going the old school forms. also you seem to not have taken care of CSRF in your forms or I might have missed that

Reply:
Took care of CSRF [here](https://github.com/waitress-andela/waitress/blob/create-admin-auth/waitress/app/templates/login.html#L5)

No reason for clinging to the old ways, there's lots of xhr request coming from the dashboard lol. I'll switch it at some point

Reply:
Feel free to go through the PR again if it's okay for approval